pkg/executor/java_options.go

```diff
@@ -279,12 +279,9 @@ func (m *appDynamicsOptions) modifyArguments(context ArgumentsContext) []string
 		args = append(args, "-Dappdynamics.analytics.agent.url="+appDynamicsAnalyticsAgentURL)
 	}
 
-	appDynamicsSplunk, exists := context.Environment("ENABLE_APPDYNAMICS_SPLUNK")
-	if exists && strings.ToUpper(appDynamicsSplunk) == "TRUE" {
-		appDynamicsHome, exists := context.Environment("HOME")
-		if exists {
-			args = append(args, "-Dappdynamics.agent.logs.dir="+appDynamicsHome+"/logs")
-		}
+	appDynamicsLogDir, exists := context.Environment("HOME")
```

Comment:
1: Variabelnavn: Variablen er "HOME", ikke appDynamicsLogDir.
2: Hva skjer hvis den ikke finnes? Den er påkrevd (http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html). 
3: Siden det er home du leter etter, kunne du brukt:

```
 usr, err := user.Current()
 if err != nil {
      log.Fatal( err )
 }
 fmt.Println( usr.HomeDir )
```

Reply:
Finnes den ikke vil den defaulte til home dir for appdynamics. Men gitt at vi skal gjøre noe fornuftig om den ikke finnes så bør dette gjøres mye tidligere?